integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -307,6 +324,277 @@ public void testUpdateDataSourceInDomainUsingAuxiliaryImage() {
+
+    final String auxiliaryImageVolumeName = "auxiliaryImageVolume1";
+    final String auxiliaryImagePath = "/auxiliary";
+
+    createSecretsForDomain(adminSecretName, encryptionSecretName, errorpathDomainNamespace);
+
+    // create stage dir for common mount image
+    Path errorpathAIPath2 = Paths.get(RESULTS_ROOT, "errorpathauxiimage2");
+    assertDoesNotThrow(() -> FileUtils.deleteDirectory(errorpathAIPath2.toFile()));
+    assertDoesNotThrow(() -> Files.createDirectories(errorpathAIPath2));
+
+    // create models dir and copy model for image
+    Path modelsPath2 = Paths.get(errorpathAIPath2.toString(), "models");
+    assertDoesNotThrow(() -> Files.createDirectories(modelsPath2));
+    assertDoesNotThrow(() -> Files.copy(
+        Paths.get(MODEL_DIR, MII_BASIC_WDT_MODEL_FILE),
+        Paths.get(modelsPath2.toString(), MII_BASIC_WDT_MODEL_FILE),
+        StandardCopyOption.REPLACE_EXISTING));
+
+    // create image with model and no wdt installation files
+    createAuxiliaryImage(errorpathAIPath2.toString(),
+        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(), errorPathAuxiliaryImage2);
+
+    // push image to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", errorPathAuxiliaryImage2, DOMAIN_IMAGES_REPO);
+      assertTrue(dockerPush(errorPathAuxiliaryImage2),
+          String.format("docker push failed for image %s", errorPathAuxiliaryImage2));
+    }
+
+    // create domain custom resource using auxiliary images
+    logger.info("Creating domain custom resource with domainUid {0} and auxiliary image {1}",
+        domainUid, errorPathAuxiliaryImage2);
+    Domain domainCR = createDomainResource(domainUid, errorpathDomainNamespace,
+        WEBLOGIC_IMAGE_NAME + ":" + WEBLOGIC_IMAGE_TAG, adminSecretName, OCIR_SECRET_NAME,
+        encryptionSecretName, replicaCount, "cluster-1", auxiliaryImagePath,
+        auxiliaryImageVolumeName, errorPathAuxiliaryImage2);
+
+    // create domain and verify it is failed
+    logger.info("Creating domain {0} with common mount image {1} in namespace {2}",
+        domainUid, errorPathAuxiliaryImage2, errorpathDomainNamespace);
+    assertDoesNotThrow(() -> createDomainCustomResource(domainCR), "createDomainCustomResource throws Exception");
+
+    // check the domain event contains the expected error msg
+    String expectedErrorMsg = "The domain resource 'spec.domainHomeSourceType' is 'FromModel'  and "
+        + "a WebLogic Deploy Tool (WDT) install is not located at  'spec.configuration.model.wdtInstallHome'  "
+        + "which is currently set to '/auxiliary/weblogic-deploy'";
+    verifyDomainEventContainsExpectedErrorMsg(timestamp, expectedErrorMsg);
+
+    // delete domain1
+    deleteDomainResource(errorpathDomainNamespace, domainUid);
+  }
+
+  /**
+   * Negative Test to create domain without domain model file, the auxiliary image contains only sparse JMS config.
```

Comment:
Add the assertion we do here to check the warning.

Reply:
fixed